cms/djangoapps/contentstore/rest_api/v1/serializers/grading.py

```diff
@@ -43,3 +46,18 @@ class CourseGradingSerializer(serializers.Serializer):
     default_grade_designations = serializers.ListSerializer(
         child=serializers.CharField()
     )
+
+
+class GradingColorSerializer(serializers.Serializer):
+    """
+    Serializer for grading color
+    """
+    color = serializers.CharField(required=True)
```

Comment:
Current UI knows nothing about the color field and won't provide any value. So for backward compatibility the field must be optional.